docs/help_classes.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Help Classes
+---
```

Comment:
Did you mean for this to be a table?

![image](https://user-images.githubusercontent.com/368767/80412141-612f0000-889b-11ea-8e56-9da43244ceeb.png)

Reply:
Docusaurus uses the table